packages/meditrak-server/src/routes/accessRequests/assertAccessRequestPermissions.js

```diff
@@ -17,10 +17,11 @@ export const assertAccessRequestPermissions = async (accessPolicy, models, acces
   }
 
   const entity = await models.entity.findById(accessRequest.entity_id);
-  if (!accessPolicy.allows(entity.country_code, TUPAIA_ADMIN_PANEL_PERMISSION_GROUP)) {
-    throw new Error('Need Admin Panel access to the country this access request is for');
+  const accessibleCountryCodes = await getAdminPanelAllowedCountryCodes(accessPolicy);
+  if (accessibleCountryCodes.includes(entity.country_code)) {
+    return true;
   }
-  return true;
+  throw new Error('Need Admin Panel access to the country this access request is for');
```

Comment:
I reckon I prefer it the old way. No logical difference right?

Reply:
Just for the slight optimisation that would give us

Reply:
Oh right...`DL`. Cool, this makes sense then. 

Could you please make sure the testers are doing performance testing as well? Not that this would make the difference - but the multiple db calls that the endpoint handlers now go through might. @Klaus-S